rust_book_ex_8_3/src/string_ex.rs

```diff
@@ -0,0 +1,21 @@
+fn starts_with_vowel(s: &String) -> bool {
+    let vowels = vec!['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+
+    if let Some(first_char) = s.chars().next() {
+        if vowels.contains(&first_char) {
+            return true;
+        } else {
+            return false;
+        }
+    } else {
+        return false;
+    }
+}
+
+pub fn get_pig_latin(s: String) -> String {
```

Comment:
It's typical for functions like this to be named as `to_*()`, as in `to_pig_latin()`.

You could also implement a trait `ToPigLatin`, so that you could do `"foo".to_pig_latin()` (standard library does this a lot with strings and chars).